lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,150 @@
+
+[.lesson-instruction]
+- Who is correct - Elise or Dani?
+** _Elise is correct. We need to multiply each value by 5._
+** _Note - it may be helpful for some students to see the Circle for @math{(5 \times 40) + (5 \times 3)}._
+
+The activities in the Launch focused on students' internalizing the structure of expressions both before and after applying the Distributive Property. During this segment of the lesson, students apply what they have learned.
+
+[.lesson-instruction]
+- Turn to @printable-exercise{true-or-false.adoc}, where you will determine if equations represented by Circles are true or false. Describe what you would change to make the false equations true.
+- Next, try @printable-exercise{wodb.adoc}. Cross out the Circle that doesn’t belong with the others, and then explain your choice.
+
+
+=== Synthesize
+
+- A common idea about arithmetic is that we must _always_ evaluate what's inside the parentheses first. Do you agree with this idea? Explain.
+- Summarize the Distributive Property in your own words.
+
+== The Distributive Property and Mental Math @duration{20 minutes}
+
+=== Overview
+
+=== Launch
+
+[.lesson-instruction]
+--
+- Look at this expression: @math{45 \times 81 - 45 \times 79}
+- Can we simplify this to an equivalent expression? How?
+** _We can apply the Distributive Property, turning this into @math{45 \times (81 - 79)}._
+- Can you represent your solving process with Circles?
+** _Invite a student to draw a Circle representation on the board. See below for a possible Circle representation._
+--
+
+[.embedded, cols="^.^5,^.^1,^.^4,^.^1,^.^3,^.^1,^.^2", grid="none", frame="none"]
+|===
+| @show{(coe '(- (* 45 81) (* 45 79)))} | &rarr; | @show{(coe '(* 45 (- 81 79)))} | &rarr; | @show{(coe '(* 45 2))} | &rarr; | 90
+|===
+
+The goal of this exercise is to help students recognize an important feature of the Distributive Property, namely:
+
+[.lesson-point]
+We can use distribution and mental math to make computation simpler.
+
+
+=== Investigate
+
+If we were asked to compute @math{70 \times 39}, we could do so using a calculator or the standard algorithm. Now that we are familiar with the Distributive Property, we have a valuable tool for efficient mental computation.
+
+[.lesson-instruction]
+- Turn to @printable-exercise{mental-math.adoc} and look at the first problem.
+- Our goal here is to make the math easier by creating an equivalent expression that we can solve in our heads! So, instead of finding the product of 70 and 39, we are going to multiply 70 by the difference of 40 and 1.
+- Complete the next Circle, which shows that we are going to _distribute_ 70.
+- We can now compute our solution by finding difference between two products.
+- Do you find this process more efficient than the standard algorithm for multiplication? Explain.
+** _Note: Without ample practice, there is a good chance that students will not find this process efficient. Like any solving strategy, it takes practice!_
```

Comment:
This might be a good opportunity to encourage teachers to have their students write 5 problems that this strategy would work well with. Alternatively those directions could be written as a challenge at the end of the workbook page using @star.